operations/Makefile

```diff
@@ -50,9 +80,32 @@ tf-03-persistent: ## Preconfigures Terraform persistent environment module
 tf-04-app: ## Preconfigures Terraform app environment module
 	${MAKE} tf-cli TF_STAGE="04-app"
 
+tf-validate: ## Validates the Terraform configuration across all modules
```

Comment:
This combined with the "don't use a backend" is great! Now anyone (even without Azure creds) can run validate super easily (otherwise it was ... less frictionless)